[2022-02-09 09:43:51,104] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: FHV_data_HW_ingestion_gcs_dag.download_dataset_task scheduled__2020-01-01T23:00:00+00:00 [queued]>
[2022-02-09 09:43:51,115] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: FHV_data_HW_ingestion_gcs_dag.download_dataset_task scheduled__2020-01-01T23:00:00+00:00 [queued]>
[2022-02-09 09:43:51,115] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:43:51,115] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:43:51,116] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:43:51,133] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-01-01 23:00:00+00:00
[2022-02-09 09:43:51,142] {standard_task_runner.py:52} INFO - Started process 1009 to run task
[2022-02-09 09:43:51,155] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'FHV_data_HW_ingestion_gcs_dag', 'download_dataset_task', 'scheduled__2020-01-01T23:00:00+00:00', '--job-id', '854', '--raw', '--subdir', 'DAGS_FOLDER/FHV_data_ingest_gcs_dag.py', '--cfg-path', '/tmp/tmp5rqc6t4t', '--error-file', '/tmp/tmphmsaxiv1']
[2022-02-09 09:43:51,157] {standard_task_runner.py:77} INFO - Job 854: Subtask download_dataset_task
[2022-02-09 09:43:51,267] {logging_mixin.py:109} INFO - Running <TaskInstance: FHV_data_HW_ingestion_gcs_dag.download_dataset_task scheduled__2020-01-01T23:00:00+00:00 [running]> on host fb702faa28b9
[2022-02-09 09:43:51,326] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:43:51,358] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=FHV_data_HW_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T23:00:00+00:00
[2022-02-09 09:43:51,360] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-09 09:43:51,363] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSL https://nyc-tlc.s3.amazonaws.com/trip+data/fhv_tripdata_2020-01.csv > /opt/***/fhv_tripdata_2020-01.csv']
[2022-02-09 09:43:51,381] {subprocess.py:85} INFO - Output:
[2022-02-09 09:43:54,375] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-09 09:43:54,477] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=FHV_data_HW_ingestion_gcs_dag, task_id=download_dataset_task, execution_date=20200101T230000, start_date=20220209T094351, end_date=20220209T094354
[2022-02-09 09:43:54,556] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:43:54,666] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-09 12:24:48,929] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: FHV_data_HW_ingestion_gcs_dag.download_dataset_task scheduled__2020-01-01T23:00:00+00:00 [queued]>
[2022-02-09 12:24:48,941] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: FHV_data_HW_ingestion_gcs_dag.download_dataset_task scheduled__2020-01-01T23:00:00+00:00 [queued]>
[2022-02-09 12:24:48,942] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 12:24:48,942] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 12:24:48,942] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 12:24:48,956] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-01-01 23:00:00+00:00
[2022-02-09 12:24:48,962] {standard_task_runner.py:52} INFO - Started process 944 to run task
[2022-02-09 12:24:48,969] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'FHV_data_HW_ingestion_gcs_dag', 'download_dataset_task', 'scheduled__2020-01-01T23:00:00+00:00', '--job-id', '1015', '--raw', '--subdir', 'DAGS_FOLDER/FHV_data_ingest_gcs_dag.py', '--cfg-path', '/tmp/tmpc6_a2fpa', '--error-file', '/tmp/tmpdtv162xm']
[2022-02-09 12:24:48,970] {standard_task_runner.py:77} INFO - Job 1015: Subtask download_dataset_task
[2022-02-09 12:24:49,042] {logging_mixin.py:109} INFO - Running <TaskInstance: FHV_data_HW_ingestion_gcs_dag.download_dataset_task scheduled__2020-01-01T23:00:00+00:00 [running]> on host 2b984638caa6
[2022-02-09 12:24:49,091] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 12:24:49,114] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=FHV_data_HW_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T23:00:00+00:00
[2022-02-09 12:24:49,115] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-09 12:24:49,116] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSL https://nyc-tlc.s3.amazonaws.com/trip+data/fhv_tripdata_2020-01.csv > /opt/***/fhv_tripdata_2020-01.csv']
[2022-02-09 12:24:49,130] {subprocess.py:85} INFO - Output:
[2022-02-09 12:24:52,617] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-09 12:24:52,648] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=FHV_data_HW_ingestion_gcs_dag, task_id=download_dataset_task, execution_date=20200101T230000, start_date=20220209T122448, end_date=20220209T122452
[2022-02-09 12:24:52,680] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 12:24:52,736] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
